docs/RELEASE.md

```diff
@@ -62,6 +62,13 @@ Cherry pick script is copied over from https://github.com/kubernetes/kubernetes/
 	make version-docs NEWVERSION=v3.7.x
 	```
 
+	This will create/update the following files and directories under `/website`: (example: https://github.com/open-policy-agent/gatekeeper/pull/1656)
```

Comment:
Can we also add something to https://github.com/open-policy-agent/gatekeeper/blob/master/.github/PULL_REQUEST_TEMPLATE.md for 
```
<!--
**Are you making changes to Gatekeeper docs?**
Gatekeeper auto-generates versioned docs. If you have any doc changes for a particular version, please update in `website/docs` as well as in `website/versioned_docs/version-[Gatekeeper version]` directory. If the change is for next release, please update in `website/docs`, then the change will be part of next versioned doc when we do a new release.
-->
```

Reply:
good call, updated